### PR TITLE
Testing splitting into 12

### PR DESCRIPTION
**Pull request by epa095**
Another test at splitting the tests, now with coverall which claims they are able to merge the results from all the 12 tests!

### CODE REVIEW

**Comment by epa095** [comment]
... And it seems to actually work! A bit annoying that when it gets the results from the first test it immediately reports it as failing with a very low test-%, but then as it receives the rest it changes its mind. So thats nice.

File: setup.cfg [hunk 1]
```diff
@@ -7,22 +7,22 @@ test             = pytest --addopts "-n auto -m 'not dockertest'"
...
-testcli          = pytest --addopts "tests/gordo_components/cli"
-testclient       = pytest --addopts "tests/gordo_components/client"
-testdataprovider = pytest --addopts "tests/gordo_components/data_provider"
-testdataset      = pytest --addopts "tests/gordo_components/dataset"
-testmodel        = pytest --addopts "tests/gordo_components/model"
-testserializer   = pytest --addopts "tests/gordo_components/serializer"
-testserver       = pytest --addopts "tests/gordo_components/server"
-testutil         = pytest --addopts "tests/gordo_components/util"
-testwatchman     = pytest --addopts "tests/gordo_components/watchman"
+testbuilder      = pytest --addopts "tests/gordo_components/builder --cov-report=xml --cov=gordo_components"
```
Comment (milesgranger) [inline setup.cfg:13]: I _think_ you would only need to at this to the `pytest.ini` where we set a bunch of the default opts.

**Comment by flikka** [comment]
@epa095 you want this PR?

**Comment by milesgranger** [comment]
@epa095 should we close this ol girl? 
 #442

**Pull request closed without merging**